Mesh_3/examples/Mesh_3/mesh_3D_weighted_image.cpp

```diff
@@ -0,0 +1,71 @@
+#include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
+
+#include <CGAL/Mesh_triangulation_3.h>
+#include <CGAL/Mesh_complex_3_in_triangulation_3.h>
+#include <CGAL/Mesh_criteria_3.h>
+
+#include <CGAL/Labeled_mesh_domain_3.h>
+#include <CGAL/Mesh_3/generate_label_weights.h>
+#include <CGAL/make_mesh_3.h>
+#include <CGAL/Image_3.h>
+#include <CGAL/IO/File_binary_mesh_3.h>
+
+// Domain
+using K = CGAL::Exact_predicates_inexact_constructions_kernel;
+using Mesh_domain = CGAL::Labeled_mesh_domain_3<K>;
+
+using Concurrency_tag =
+#ifdef CGAL_CONCURRENT_MESH_3
+  CGAL::Parallel_tag;
+#else
+  CGAL::Sequential_tag;
+#endif
+
+// Triangulation
+using Tr   = CGAL::Mesh_triangulation_3<Mesh_domain, CGAL::Default, Concurrency_tag>::type;
+using C3t3 = CGAL::Mesh_complex_3_in_triangulation_3<Tr>;
```

Comment:
```suggestion
// Triangulation
using Concurrency_tag = CGAL::Parallel_if_available_tag;
using Tr   = CGAL::Mesh_triangulation_3<Mesh_domain, CGAL::Default, Concurrency_tag>::type;
using C3t3 = CGAL::Mesh_complex_3_in_triangulation_3<Tr>;
```

See https://doc.cgal.org/latest/STL_Extension/structCGAL_1_1Parallel__if__available__tag.html